source/manual/pagerduty.html.md

```diff
@@ -1,13 +1,29 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: PagerDuty drill
-parent: "/manual.html"
+title: PagerDuty
+section: 2nd line
+type: learn
 layout: manual_layout
-section: Icinga alerts
-last_reviewed_on: 2019-07-18
-review_in: 6 months
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 3 months
 ---
 
+Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming
```

Comment:
I wonder if we want to add something to this page about it's the responsibility of the developers to make sure their details are up to date in PagerDuty and they're correctly scheduled in.